Cargo.lock

```diff
@@ -2895,18 +2895,18 @@ checksum = "ff011a302c396a5197692431fc1948019154afc178baf7d8e37367442a4601cf"
 
 [[package]]
 name = "openssl-src"
-version = "111.27.0+1.1.1v"
+version = "300.1.6+3.1.4"
```

Comment:
This, for me, means we switch from OpenSSL 1.1.1v to 3.1.4.
Do we understand the implications fully? Asking before we merge this.

https://github.com/alexcrichton/openssl-src-rs
> This crate follows the latest minor and patch versions for each maintained major version, according to the [OpenSSL release strategy](https://www.openssl.org/policies/releasestrat.html). It has no specific support for LTS versions.
> 
> The crate versions follow the X.Y.Z+B pattern:
> 
>     The major version X is the upstream OpenSSL API/ABI compatibility version:
>         300 for 3.Y.Z
>     The minor Y and patch Z versions are incremented when making changes to the crate, either OpenSSL update or internal changes.
>     B contains the full upstream OpenSSL version, like 1.1.1k or 3.0.7. Note that this field is actually ignored in comparisons and only there for documentation.

Reply:
Why doing this in this unrelated PR though?

Reply:
It's not unrelated, it's a transitive dependency that got updated. The PR updated `openssl` from `0.10.56` to `0.10.60`, newer version of `openssl` requires newer version of `openssl-sys`, so it got updated from `0.9.91` to `0.9.96`, and newer version of `openssl-sys` requires newer version of `openssl-src`, so it got updated from `111.27.0+1.1.1v` to `300.1.6+3.1.4`.

Since the underlying version of OpenSSL is an implementation detail and does not change the API and behavior of the higher-level Rust wrapper in any breaking way, this is permitted by semver: a patch update of a library can update a major version of its dependency if it's an implementation detail.